test/gtest/uct/test_md.cc

```diff
@@ -778,6 +778,7 @@ UCS_TEST_SKIP_COND_P(test_md, dereg_bad_arg,
         ASSERT_UCS_STATUS_EQ(UCS_ERR_UNSUPPORTED, status);
 
         params.field_mask = UCT_MD_MEM_DEREG_FIELD_MEMH;
+        /* coverity[double_free] */
```

Comment:
why needed in this PR?

Reply:
Coverity gives false positives without it, I discovered when initially wrote this PR.
Without these suppressions, it thinks that the invalid `uct_md_mem_dereg_v2` calls really free the `memh`, thus it is "freed multiple times".